packages/components/src/toggle-group-control/toggle-group-control/styles.ts

```diff
@@ -39,7 +39,7 @@ export const toggleGroupControl = ( {
 		content: '';
 		position: absolute;
 		pointer-events: none;
-		background: ${ COLORS.gray[ 900 ] };
+		background: ${ COLORS.theme.foreground };
```

Comment:
This actually fixes the active state with dark theme:

|Before|After|
|---|---|
|![Screenshot 2024-12-09 at 11 49 52](https://github.com/user-attachments/assets/44d68570-bea2-4ea2-8184-01c9b67583b1)|![Screenshot 2024-12-09 at 11 49 37](https://github.com/user-attachments/assets/103ca3c1-d876-49f6-8965-024396cdd7ac)|